app.js

```diff
@@ -6,9 +6,26 @@ const app = express();
 // SERVER HEALTHCHECK
 app.get("/api/healthcheck", healthcheck);
 
+
+//ENDPOINTS MIDDLEWARE
+app.get("/api/topics", getTopics);
+
+//ERROR HANDLING
+
+/////////////BAD REQUEST
+app.use((err, req, res, next) => {
```

Comment:
This isn't needed right now. At the moment you bad URLs test is passing as express is returning a 404 for you